pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc_store {
+    //index-client
+    rpc BatchWrite(WriteBatch) returns (Nothing);
+    rpc QueryPages(IndexQuery) returns (stream ReadBatch);
+
+    //storage-client
+    rpc PutChunks(Chunks) returns (Nothing);
+    //In GetChunks rpc request send buf as nil
+    rpc GetChunks(Chunks) returns (stream Chunks);
```

Comment:
I'm not sure `Chunks` is the right data structure to use here. Design wise, it would be cleaner if we would have a different data structure for read and write. In the read path (this rpc) what you actually need is a list of chunk keys (table name and key). In the write path you need the chunk key + data.

Reply:
Initially, we maintained different structures but @gouthamve, and I felt to use the same data structure for read & write path to make it simple. Maybe @gouthamve can here on the context behind this decision.

Reply:
Design wise I think it's a bit risky for future evolutions. As I commented in the review message, I think that having a dedicated Request and Response message for each function is more future proof, ie.

```
rpc PutChunks(PutChunksRequest) returns (google.protobuf.Empty);
rpc GetChunks(GetChunksRequest) returns (stream GetChunksResponse);
```